db/schema.rb

```diff
@@ -10,14 +10,24 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_01_21_210300) do
+ActiveRecord::Schema.define(version: 2019_01_25_190049) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
```

Comment:
I prefer [this options](https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals) for `single`/`double` quotes. If you have another opinion, please, write `cop` to `.rubocop.yml`

Reply:
So do I, but this file is generated automatically and you will have to change quotes after each migration.